.claude/skills/guard-maturity/scripts/run_report.py

```
@@ -0,0 +1,139 @@
+    "Mature": 4,
+    "Proficient": 5,
+}
+
+
+def _run_aggregate(known_guards: list[str], source: str | None) -> dict:
+    cmd = [sys.executable, str(AGGREGATE)]
+    for g in known_guards:
+        cmd.extend(["--guard", g])
+    if source:
+        cmd.extend(["--source", source])
+    proc = subprocess.run(cmd, capture_output=True, text=True, check=False)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)